README.md

```diff
@@ -5,6 +5,9 @@
 This repository consists of tooling to manage Firebase rules, indexes, etc. and
 lambda functions deployed to Netlify to manage the data collected in Firebase.
 
+## Code of Conduct
+All participants are expected to treat others with respect and follow our [Code of Conduct](https://github.com/lansingcodes/www/blob/master/.github/CODE_OF_CONDUCT.md).
```

Comment:
I think here we could also link to https://www.lansing.codes/code-of-conduct/ 

Would you mind updating the link here and in `CONTRIBUTING.md` to match, then remove `CODE_OF_CONDUCT.md`?

Reply:
Thank you Erik. I made the changes. 

Do you think in all repos, code-of-conduct should link to https://www.lansing.codes/code-of-conduct/ ? Please advise.

At present in "www" repo, its pointing to https://github.com/lansingcodes/www/blob/master/.github/CODE_OF_CONDUCT.md

Reply:
Yeah, after thinking about it, it makes sense that we should have one Code of Conduct as our "source of truth" where we direct people and the one visible in the `www` project should be it.